components/tailscale/component_output.go

```diff
@@ -122,12 +151,28 @@ func Get(ctx context.Context) (_ any, e error) {
 		}
 	}()
 
-	ver, err := CheckVersion()
-	if err != nil {
-		return nil, err
+	var ver *VersionInfo
+	for i := 0; i < 5; i++ {
```

Comment:
retry on version check makes very little sense... why would this fail... if this fails, let us just wait for the next run to succeed. version check failure should be just a warning anyway. we should check tailscale systemd status.

Reply:
@cardyok Can we just drop tailscale version check here?

https://github.com/leptonai/gpud/blob/ab4037114adede58f63624c82040471deb513dc6/manager/controllers/package_controller.go#L99-L107

already does tailscale version checks

Reply:
yes we can, actually package controller already checking status/version of kubelet/containerd/tailscale